configure.ac

```diff
@@ -496,7 +496,7 @@ if test "${enable_dbengine}" != "no" -a "${UV_LIBS}" -a "${LZ4_LIBS}" -a "${JUDY
     OPTIONAL_JUDY_CFLAGS="${JUDY_CFLAGS}"
     OPTIONAL_JUDY_LIBS="${JUDY_LIBS}"
     OPTIONAL_SSL_CFLAGS="${SSL_CFLAGS}"
-    if test "${enable_wolfssl}" = "yes"; then
+    if test "${WOLFSSL_LIBS}"; then
```

Comment:
It means that if both libraries are installed, wolfSSL is preferred.

Reply:
You can just check the OpenSSL first.

Reply:
I apoligize! I misunderstood what you said :eyes: .
I push another commit adding an `and`, as you said OpenSSL is preferred and we need to use `WolfSSL` libs only when the option `--use-wolfssl` is used.